process/block/metablock.go

```diff
@@ -298,7 +298,7 @@ func (mp *metaProcessor) ProcessBlock(
 	}
 
 	if !bytes.Equal(validatorStatsRH, header.GetValidatorStatsRootHash()) {
-		log.Debug("Validator stats root hash does not match", "validatorStatsRH", validatorStatsRH,
+		log.Warn("Validator stats root hash does not match", "validatorStatsRH", validatorStatsRH,
```

Comment:
no one cares about this processing error. Might leave it at debug level Might rephrase it like
```GO
log.Debug("validator stats root hash mismatch", 
"computed", validatorStatsRH,
"received", header.GetValidatorStatsRootHash(),
)
```

Reply:
done.